repository/src/test/java/org/alfresco/repo/serviceaccount/ServiceAccountRegistryImplTest.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * #%L
+ * Alfresco Data model classes
```

Comment:
I think we should use the 'Repository' copyright header here instead of the Alfresco Data Model one